src/main/java/io/github/fusionflux/portalcubed/mixin/client/ClientLevelMixin.java

```diff
@@ -45,6 +49,15 @@ private void init(CallbackInfo ci) {
 		this.portalManager = new ClientPortalManager((ClientLevel) (Object) this);
 	}
 
+	@WrapOperation(method = "tickNonPassenger", at = @At(value = "INVOKE", target = "Lnet/minecraft/world/entity/Entity;tick()V"))
+	private void disintegrationTick(Entity instance, Operation<Void> original) {
```

Comment:
ticking is weird, make sure passengers disintegrate properly

Reply:
They don't, but disintegration makes them not passengers (`stopRiding` gets called)

Reply:
what about something starting riding after fizzling